src/main/java/com/sforce/cd/apexUnit/client/QueryConstructor.java

```diff
@@ -48,9 +48,10 @@
 	 * 
 	 * @return - Query to fetch apex classes as String
 	 */
-	public static String generateQueryToFetchApexClassesBasedOnRegex(String regex) {
+	public static String generateQueryToFetchApexClassesBasedOnRegex(String namespace, String regex) {
```

Comment:
We are just checking for default namespace(null)  for now , correct?
Have we added namespace parameter for future extensibility?